src/NosCore.Parser/Parsers/ItemParser.cs

```diff
@@ -0,0 +1,679 @@
+
+                    item.Element = elementdic.First().Key;
+                    if (elementdic.First().Value != 0 && elementdic.First().Value == elementdic.ElementAt(1).Value)
+                    {
+                        item.SecondaryElement = elementdic.ElementAt(1).Key;
+                    }
+                }
+            }
+            SetVehicles(items, new Dictionary<byte, List<(short, short)>>
+            {
+                { 20, new List<(short, short)> { (9054, 2368), (1906, 2368), (9055, 2370), (1907, 2370), (9058, 2406), (1965, 2406), (9065, 2411),
+                    (5008, 2411), (5238, 1817), (5226, 1817), (5240, 1819), (5228, 1819), (5234, 2522), (5236, 2524) } },
+                { 21, new List<(short, short)> { (9070, 2429), (5117, 2429), (9073, 2432), (5152, 2432), (5196, 2517), (9078, 2520), (5232, 2520),
+                    (5321, 2528), (9090, 2934), (5386, 2934), (9091, 2936), (5387, 2936), (9092, 2938), (5388, 2938), (9093, 2940), (5389, 2940),
+                    (9094, 2942), (5390, 2942), (5391, 2944), (9115, 3679), (5997, 3679), (9079, 2522), (9080, 2524), (9081, 1817), (9082, 1819) } },
+                { 22, new List<(short, short)> { (9083, 2526), (5319, 2526), (5323, 2530), (9086, 2928), (5330, 2928), (9088, 2932), (5360, 2932), (9084, 2528), (9085, 2930) } },
+                { 14, new List<(short, short)> { (9087, 2930), (5332, 2930), (5914, 2513) } },
+                { 16, new List<(short, short)> { (5173, 2511) } },
+            });
+
+            _itemDao.InsertOrUpdate(items);
+            _bcardDao.InsertOrUpdate(items.Where(s => s.BCards != null).SelectMany(s => s.BCards));
+
+            _logger.Information(LogLanguage.Instance.GetMessageFromKey(LogLanguageKey.ITEMS_PARSED), items.Count);
+        }
+
+
+        private List<BCardDto> ImportBCards(Dictionary<string, string[][]> chunk)
+        {
+            var list = new List<BCardDto>();
+            for (var i = 0; i < 5; i++)
+            {
+                var type = (byte)int.Parse(chunk["BUFF"][0][2 + 5 * i]);
+                if ((type == 0) || (type == 255))
+                {
+                    continue;
+                }
+
+                var first = int.Parse(chunk["BUFF"][0][3 + 5 * i]);
+                var comb = new BCardDto
+                {
+                    ItemVNum = Convert.ToInt16(chunk["VNUM"][0][2]),
+                    Type = type,
+                    SubType = (byte)((int.Parse(chunk["BUFF"][0][5 + 5 * i]) + 1) * 10 + 1),
+                    IsLevelScaled = Convert.ToBoolean(first % 4),
+                    IsLevelDivided = (uint)(first < 0 ? 0 : first) % 4 == 2,
+                    FirstData = (short)(first > 0 ? first : -first / 4),
+                    SecondData = (short)(int.Parse(chunk["BUFF"][0][4 + 5 * i]) / 4),
+                    ThirdData = (short)(int.Parse(chunk["BUFF"][0][6 + 5 * i]) / 4)
+                };
+                list.Add(comb);
+            };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4456396)